### ISSUE TITLE
The ability to show sparsely defined  information in the 2D canvas

### ISSUE THREAD

**Issue by FredrikKarlssonSpeech** [IPS-LMU/EMU-webApp#294]
It would be very nice to be able to show information that is available for particular portions of the signal only, but may well be very multidimensional, in the 2D canvas in an efficient way. You could do it now by converting to SSFF, but that would involve the transfer of lots of zeros to the web app only to show some useful information. The use case I envision is the visualization of some of the features computed to identify the speaker in a dialogue or a full voice analysis for a prolonged vowel only.

In this case, it would be great to force the information into an SSFF but be able to transfer a structure like

```
  [
     {
   "start_sample": 0,
  "end_sample": 44100.
  "data": 
        [ "datapoint 1": <number>, 
     "datapoint 2": <number>,
       ....
      ]
    },
     {
   "start_sample": 150000,
  "end_sample":  300000.
  "data": 
        [ "datapoint 1": <number>
           , "datapoint 2": <number>,
        ...
       ]
    },
   ...
]
```

and be able to select  "datapoint 400",...  "datapoint 600" for display in the 2D panel in the manner that the user set up,  when defined for the current cursor position. 

(Sorry for the hand-coded JSON. I am sure it is wrong.)

**Comment by raphywink** [comment]
Yeah I can see how that might be a nice to have feature. Will keep this issue open (don't know when I'll get around to looking at is more closely)
